lace_circular_ground.inx

```diff
@@ -37,7 +37,7 @@
         <option value="pt">pt</option>
         </param>
     <param name="lineheading" type="description">Color:</param>
-    <param name="linecolor" type="color">255</param>
+    <param name="linecolor" type="color" _gui-text="line color">255</param>

```

Comment:
Color selection works for only 2 out of the 5 extions
![image](https://user-images.githubusercontent.com/10553630/81497220-54f46b00-92bd-11ea-8a70-1c8b577e3130.png)

Reply:
All extensions show color for me.  Make sure you replace the inx files as well as the py files.  Some inx files were missing the _gui-text information.

I have now changed the color selection to a color button in all extensions.  The look is less busy and the interface is easier for novice users.

![color](https://user-images.githubusercontent.com/259897/81500700-e7871100-92a1-11ea-9837-99a451efdb2e.png)

Reply:
I tested on mac without ever having InkScape on that system before. But now it's looking OK.

The new color selection is great 👍 

I'll create new screenshots for https://d-bl.github.io/inkscape-bobbinlace/ before merging.

Reply:
so glad you decided to update. thanks.